public/app/core/logs_model.ts

```diff
@@ -260,7 +260,9 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel {
       const timeLocal = time.format('YYYY-MM-DD HH:mm:ss');
       const timeUtc = toUtc(ts).format('YYYY-MM-DD HH:mm:ss');
 
-      const message = stringField.values.get(j);
+      let message = stringField.values.get(j);
+      // This should be string but sometimes isn't (eg elastic) because the dataFrame is not strongly typed.
+      message = typeof message === 'string' ? message : JSON.stringify(message);
```

Comment:
Added it here in logs transform instead of in the logRow component cause here the typing says message should be string and it would probably break much more if it wasn't. For tableModel the row is `any[]` so I assume the handling there is different and there is less code relying on it of being some specific type.